lib/calendarium-romanum/enums.rb

```diff
@@ -28,9 +28,10 @@ class Colours < Enum
   Colors = Colours
 
   class Season
-    def initialize(symbol, colour)
+    def initialize(symbol, colour, name)
```

Comment:
The rest is not correct. The variable you set here isn't used anywhere. And it's not needed: method `Season#name` exists already - see below.  Season name cannot be simply set as a String in code, because calendarium-romanum is internationalized and this is one of the values which need to be translated. It's why each season has instance variable `@symbol` (holding a Symbol which serves as internal identifier of the season), which is used in `#name` to retrieve translated season name for the current locale from `i18n`.